app/controllers/medicaid/contact_social_security_controller.rb

```diff
@@ -1,5 +1,11 @@
 module Medicaid
   class ContactSocialSecurityController < Medicaid::ManyMemberStepsController
+    def step
+      @step ||= step_class.new(
+        members: members_requesting_health_insurance,
```

Comment:
Now we can only add members to the step that actually need updating! Prevents no-ops for irrelevant members in the update step.